BFAIR/FIA_MS/FIA_MS_tools.py

```diff
@@ -0,0 +1,94 @@
+import numpy as np
+import pandas as pd
+from pyopenms import FeatureMap, FeatureXMLFile
+
+__version__ = "0.0.1"
+
+
+def extractNamesAndIntensities(feature_dir, sample_names, database):
```

Comment:
Can you add the docstrings for this method?

Reply:
Oops, forgot about that